configs/centerpoint/argoverse_centerpoint_pp_02voxel_circle_nms.py

```diff
@@ -0,0 +1,192 @@
+import itertools
+import logging
+
+from det3d.builder import build_box_coder
+from det3d.utils.config_tool import get_downsample_factor
+
+norm_cfg = None
```

Comment:
is there a particular reason we are using this config, over all the other configs listed here (https://github.com/hchittanuru3/CenterPoint/tree/328317fc3dbb0d55f1f89264e10fed2a59fdf226/configs/centerpoint)

Was this one the top-performing?

Reply:
The filename means PointPillars + voxel size of 0.2 by 0.2 + circle NMS.
It appears that this is the best performing among all other configs/architectures, because this is the one they decided to use for demo.